test/ui/cypress/support/cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
```

Comment:
Why is this file here, this is not the right place for a config file and we already have one in the right place. Please remove.

Reply:
@tznamena , I am not sure why this file exists in this folder. It seems this file was auto-generated when I ran the automated test scenarios. I will double check it.

Reply:
:+1: